adapters/triplelift/triplelifttest/exemplary/simple-video.json

```diff
@@ -1,114 +1,144 @@
 {
-	"mockBidRequest": {
```

Comment:
why is this test in particular so different?

Reply:
It's not, VSCode just auto-formatted the JSON indents to 2 spaces 🤷‍♂

Reply:
can you ./validate.sh

Reply:
that will format the ws in the standard way